### ISSUE TITLE
How to calculate model parameters? 

### ISSUE THREAD

**Issue by HATFormer** [VMarsocci/3DCD#2]
Dear authors,
Thanks so much for your impressive contribution! The paper is convincing and the code is smooth for reading.
But I do not know how to calculate the model parameters as stated in your paper -- 13.1M.
I tried to use the 'thop' python library, it is easy for using. Specifically, to add
`from thop import profile`
`flops, params = profile(net,inputs=[torch.cat([t1,t2],dim=0).cuda()])`
`print('FLOPs = ' + str(flops / 1000 ** 3) + 'G')`
`print('Params = ' + str(params / 1000 ** 2) + 'M')`
in Line201 of test.py 
And slightly modified the forward function of MTBIT.py in Line195 as 
`def forward(self, x1x2):`
`        bsize = x1x2.shape[0]//2`
`        x1, x2 = x1x2[:bsize,::],x1x2[bsize:,::]`
But the output is 
`FLOPs = 9.652185088G`
`Params = 3.111875M`
instead of 13.1M. So it must be something wrong of my implementation.
Can you tell me the method you calculating the model parameters? Thanks so much!

**Comment by VMarsocci** [comment]
Hello sorry for my late answer, for the number of parameters we used the function that you can find in `train.py`, named `count_parameters`. Thanks for your question :)
